packages/f-checkout/test-utils/component-objects/f-checkout.component.js

```diff
@@ -21,7 +25,14 @@ exports.inputs = {
 exports.waitForCheckoutComponent = () => checkoutComponent().waitForExist();
 exports.isCheckoutComponentDisplayed = () => checkoutComponent().isDisplayed();
 exports.isAllergenLinkDisplayed = () => allergenLink().isDisplayed();
+exports.isDropdownDisplayed = () => dropdownSelector().isDisplayed();
 
+exports.selectOption = () => $$("select option");
+exports.showDropdownOption = (index) => {
```

Comment:
The name of this function doesn't seem quite right, maybe something like `getDropdownOptionText`?